src/javascript/app/Modules/Trading/Components/Form/TradeParams/Duration/duration-wrapper.jsx

```diff
@@ -48,6 +48,21 @@ class DurationWrapper extends React.Component {
         }
     }
 
+    getSetDurationMinMax = (duration_min_max, contract_expiry_type, duration_unit) => {
```

Comment:
why not `getDurationMinMaxValues`?

Reply:
@cakasuma pls check Zaki's comment before I merge this

Reply:
imo split it into two functions, one for getting the min max and one for setting min max to state since you don't need the return values on line 128

(returning an array would be cleaner than an object I think also)